### ISSUE TITLE
Enhance Query Engine

### ISSUE THREAD

**Issue by tudorian** [SAP/PyRFC#367]
Hello,

It is possible to use another SAP table as a reference when extraction data from the current tables

Eg: 
SELECT * from MARC INNER JOIN MARA ON MARC~MATNR = MARA~MATNR
WHERE MARA~MTART = 'ABC'

Thanks,

Tudor

**Comment by amarvin** [comment]
I don't think you can do joins using `pyrfc`, unless you call an RFC that does the tables joins on the SAP-side. Rather than doing the join on the SAP-side, you could do query the data from the two tables separately and then inner join them on your client-side.

Note that where conditions are a bit tough to use with the basic `pyrfc`. I developed https://github.com/amarvin/pyrfc-read to make this easier!
